src/amo/components/AddAddonToCollection/index.js

```diff
@@ -24,19 +24,26 @@ import type { DispatchFunc } from 'core/types/redux';
 import type { CollectionType } from 'amo/reducers/collections';
 import type { AppState } from 'amo/store';
 import type { ElementEvent } from 'core/types/dom';
+import type { ReactRouterType } from 'core/types/router';
 
 import './styles.scss';
 
 type Props = {|
-  _window: typeof window | Object,
   addon: AddonType | null,
+|};
+
+type InternalProps = {|
+  ...Props,
   // These are all user collections that the current add-on is a part of.
   addonInCollections: Array<CollectionType> | null,
   className?: string,
```

Comment:
It doesn't look like `className` is internal.

Reply:
it does not look like it is used actually :p